x-pack/plugins/security_solution/public/timelines/components/formatted_ip/index.tsx

```diff
@@ -143,6 +162,36 @@ const AddressLinksItemComponent: React.FC<AddressLinksItemProps> = ({
     [address, contextId, eventId, fieldName]
   );
 
+  const dispatch = useDispatch();
+  const eventContext = useContext(StatefulEventContext);
+
+  const openNetworkDetailsSidePanel = useCallback(() => {
+    if (address && eventContext?.timelineID && eventContext?.tabType) {
+      const { tabType, timelineID } = eventContext;
+      const updatedExpandedDetail: TimelineExpandedDetailType = {
+        panelView: 'networkDetail',
+        params: {
+          ip: address,
+          flowTarget: fieldName.includes('destination') // TODO: does this matter here?
```

Comment:
@XavierM Do I need to make this distinction?

Reply:
I am fine with it, if we do that
```suggestion
          flowTarget: fieldName.includes(FlowTarget.destination)
```

Reply:
done